src/DrTests-TestsProfiling/DTTestsProfilerVisitor.class.st

```diff
@@ -33,12 +33,13 @@ DTTestsProfilerVisitor >> visitTestCase: aTestCase [
 	"Defines the behaviour while visiting a TestCase.
 	 This method must be overriden by concrete subclasses.
 	"
-
+	"We make explicit the dependency to duration"
+	
 	| aTestresult |
 	self profilingData
 		add:
 			(DTTestCaseProfilingData
 				testCase: aTestCase
-				duration: [ aTestresult := aTestCase run ] timeToRun
+				duration: (Duration milliSeconds: ([ aTestresult := aTestCase run ] millisecondsToRun))
```

Comment:
I wanted to deprecate timeToRun and to make explicit the dependency from DrTest to the future Duration package.

Reply:
But I like timeToRun!!! Just not in the kernel!

Reply:
Here it was really syntactical sugar and make the dependency analysis more difficult. 
This is why I switch it. And it does not remove the dependency of DrTest to Duration.

Reply:
I agree with guille in timeToRun

Reply:
Me too. Now I repeat it because may be I was unclear. 
In DrTest we can have a dependency to Duration but 
- I wanted to make this explicit hence I refer to the class Duration
- I wanted to make sure that we do not use timeToRun in BlockClosure.